src/api/ml/z3native_stubs.c.pre

```diff
@@ -410,25 +411,26 @@ MK_CTX_OF(ast, 16) // let's say 16 bytes per ast
                                                                         \
   MK_CTX_OF(X, USED)
 
-MK_PLUS_OBJ_NO_REF(symbol, 32)
-MK_PLUS_OBJ_NO_REF(constructor, 32)
-MK_PLUS_OBJ_NO_REF(constructor_list, 32)
-MK_PLUS_OBJ_NO_REF(rcf_num, 32)
-MK_PLUS_OBJ(params, 128)
-MK_PLUS_OBJ(param_descrs, 128)
-MK_PLUS_OBJ(model, 512)
-MK_PLUS_OBJ(func_interp, 128)
-MK_PLUS_OBJ(func_entry, 128)
-MK_PLUS_OBJ(goal, 128)
-MK_PLUS_OBJ(tactic, 128)
-MK_PLUS_OBJ(probe, 128)
-MK_PLUS_OBJ(apply_result, 128)
-MK_PLUS_OBJ(solver, 20 * 1000 * 1000) // pretend a solver is 20MB
-MK_PLUS_OBJ(stats, 128)
-MK_PLUS_OBJ(ast_map, 1024 * 2)
-MK_PLUS_OBJ(ast_vector, 128)
-MK_PLUS_OBJ(fixedpoint, 20 * 1000 * 1000)
-MK_PLUS_OBJ(optimize, 20 * 1000 * 1000)
+// FUDGE
```

Comment:
vanilla size?

Reply:
it's something I stole from Jasmin B, it's just to indicate a potential tweaking point for heuristics :).